src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -126,6 +126,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('force', help='Overwrite the existing tag of the image to be imported.', action='store_true')
         c.argument('no_wait', help="Do not wait for the import to complete and return immediately after queuing the import.", action='store_true')
 
+    with self.argument_context('acr config authentication-as-arm') as c:
+        c.argument('registry_name', options_list=['--registry', '-r', c.deprecate(target='-n', redirect='-r', hide=True), c.deprecate(target='--name', redirect='--registry', hide=True)])
+        c.argument('status', help="Indicates whether authentication-as-arm is enabled.", arg_type=get_enum_type(PolicyStatus))
```

Comment:
Please use the first-person voice in the help message
```suggestion
        c.argument('status', help="Indicate whether authentication-as-arm is enabled.", arg_type=get_enum_type(PolicyStatus))
```

Reply:
done